packages/ethtool/ethtool.spec

```diff
@@ -0,0 +1,33 @@
+Name:           %{_cross_os}ethtool
+Epoch:          2
+Version:        6.2
+Release:        1%{?dist}
+Summary:        Settings tool for Ethernet NICs
+License:        GPL-2.0-only AND GPL-2.0-or-later
+URL:            https://www.kernel.org/pub/software/network/ethtool/
+Source0:        https://www.kernel.org/pub/software/network/ethtool/ethtool-%{version}.tar.xz
+BuildRequires:  %{_cross_os}glibc-devel
+BuildRequires:  %{_cross_os}libmnl-devel
+Conflicts:      filesystem < 3
```

Comment:
You don't need this

Reply:
I don't need the `Conflicts` but the others are needed right?

Reply:
Yes, other are needed but `Conflicts` isn't